### PR TITLE
Update configuration for helm

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Helm configuration for zipkin plugin

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/conf/traces/traces.conf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<source>
+    @type zipkin
+    port 9411
+</source>
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/conf/traces/traces.conf]: Labeling required. As tracing is last configuration part, previous `<match **>` catches all tracing traffic

File: deploy/helm/sumologic/conf/traces/traces.conf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<filter tracing.**>
+    @type stdout
+</filter>
+<match tracing.**>
+    @type zipkin
+    endpoint "#{ENV['SUMO_ENDPOINT_TRACES']}"
+    content_type application/json
+    overwrite_content_type application/vnd.sumologic.zipkin
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/conf/traces/traces.conf]: `overwrite_content_type` -> `overwrite_content_type` or change this value in zipkin plugin

File: deploy/helm/sumologic/conf/traces/traces.conf [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<source>
+  @type zipkin
+  @label @TRACING
+  port 9411
+</source>
+<label @TRACING>
+  <filter tracing.**>
+    @type stdout
+  </filter>
```
Comment (frankreno) [inline deploy/helm/sumologic/conf/traces/traces.conf]: By doing this, fluentd will log the traces to STDOUT right? I assume you are doing this for debugging but keep the challenge is we collect the fluentd logs in the K8s collection process and therefore customers will have an ingest spike for this.  If so, we should probably make this configurable and not on by default.

File: deploy/helm/sumologic/values.yaml [hunk 4]
```diff
@@ -89,6 +89,10 @@ sumologic:
     # Source category for the Events source. Default: "{clusterName}/events"
     sourceCategory: ""
 
+  # Traces configuration
```
Comment (frankreno) [inline deploy/helm/sumologic/values.yaml]: Can we make sure we add a comment indicating this is an alpha/closed beta feature and not intended for use by everyone? Probably work with Pawel on the verbage.

File: deploy/helm/sumologic/conf/traces/traces.conf (hunk 1 above)
Comment (samjsong) [reply deploy/helm/sumologic/conf/traces/traces.conf]: cc @vsinghal13 , maybe we should update all of our sources to use labels to make clear distinction between logs, metrics, and traces. when we added the `logs.source.generic.conf`, we didn't take into consideration non-log data when we added the `<match **>`

File: deploy/helm/sumologic/values.yaml (hunk 4 above)
Comment (vsinghal13) [reply deploy/helm/sumologic/values.yaml]: +1

File: deploy/helm/sumologic/conf/traces/traces.conf (hunk 1 above)
Comment (sumo-drosiek) [reply deploy/helm/sumologic/conf/traces/traces.conf]: I created jira task for it: SUMO-130461

File: deploy/helm/sumologic/templates/configmap.yaml [hunk 5]
```diff
@@ -10,6 +10,12 @@ data:
...
     @include metrics.conf
     @include logs.conf
+{{- if .Values.sumologic.traces.enable }}
+    @include traces.conf
+{{- end }}
   {{- (tpl (.Files.Glob "conf/*.conf").AsConfig .) | nindent 2 }}
   {{- (tpl (.Files.Glob "conf/metrics/*.conf").AsConfig .) | nindent 2 }}
   {{- (tpl (.Files.Glob "conf/logs/*.conf").AsConfig .) | nindent 2 }}
+  {{- if .Values.sumologic.traces.enable }}
+  {{- (tpl (.Files.Glob "conf/traces/*.conf").AsConfig .) | nindent 2 }}
```
[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply deploy/helm/sumologic/templates/configmap.yaml]: Jira task for revisiting: SUMO-130462

**Comment by sumo-drosiek** [comment]
Merging with squash to the `drosiek-zipkin-plugin`. `drosiek-zipkin-plugin` is going to be merged to the master when CI success and all approval gathered :)

**Pull request merged**